CatProducer/python/pileupWeight_cff.py

```diff
@@ -610,14 +610,65 @@
+2.337951e+00,2.692769e+03,1.835026e+04,2.925740e+04,3.458540e+04,
+5.303284e+04,4.018890e+04,1.112240e+05,1.781261e+05,2.975992e+05,
+6.126548e+05,1.446731e+06,4.475962e+06,1.205399e+07,2.094242e+07,
+2.704624e+07,3.101324e+07,3.199685e+07,2.791044e+07,2.043488e+07,
+1.345124e+07,8.477045e+06,5.085139e+06,2.783320e+06,1.346012e+06,
+5.678282e+05,2.104109e+05,7.213909e+04,2.722495e+04,1.485085e+04,
+1.166367e+04,1.047979e+04,9.583685e+03,8.721376e+03,7.931882e+03,
+7.272196e+03,6.766851e+03,6.407705e+03,6.165121e+03,6.000496e+03,
+5.874260e+03,5.750384e+03,5.611628e+03,5.444083e+03,5.241055e+03,
+5.001617e+03,4.728756e+03,4.427898e+03,4.105816e+03,3.769852e+03,
+),
+#pileupCalc.py -i Cert_271036-273450_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt --inputLumiJSON pileup_latest.txt --calcMode true --minBiasXsec 72450 --maxPileupBin 50 --numPileupBins 50 PileUpData_Up.root 
+"Cert_271036-273450_13TeV_PromptReco_Collisions16_JSON_NoL1T_Up":cms.vdouble(
+2.201232e+00,1.505004e+03,1.675106e+04,2.555822e+04,2.980619e+04,
+5.243856e+04,3.424635e+04,7.714873e+04,1.454256e+05,2.163806e+05,
+4.004034e+05,8.333628e+05,2.115136e+06,6.327284e+06,1.425730e+07,
+2.174921e+07,2.672045e+07,2.996876e+07,3.029785e+07,2.614669e+07,
+1.928275e+07,1.293888e+07,8.348181e+06,5.163220e+06,2.951071e+06,
+1.512285e+06,6.852224e+05,2.748410e+05,1.004227e+05,3.715798e+04,
+1.753842e+04,1.214566e+04,1.051477e+04,9.628343e+03,8.829922e+03,
+8.062598e+03,7.375578e+03,6.809661e+03,6.377986e+03,6.069518e+03,
+5.858054e+03,5.711482e+03,5.597296e+03,5.485310e+03,5.361487e+03,
+5.213819e+03,5.036009e+03,4.826540e+03,4.587247e+03,4.322140e+03,
+),
+#pileupCalc.py -i Cert_271036-273450_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt --inputLumiJSON pileup_latest.txt --calcMode true --minBiasXsec 68827 --maxPileupBin 50 --numPileupBins 50 PileUpData_Dn.root 
+"Cert_271036-273450_13TeV_PromptReco_Collisions16_JSON_NoL1T_Dn":cms.vdouble(
+2.345262e+00,2.763278e+03,1.841499e+04,2.944567e+04,3.494226e+04,
+5.288455e+04,4.084824e+04,1.130166e+05,1.800750e+05,3.030854e+05,
+6.268655e+05,1.493732e+06,4.654222e+06,1.240208e+07,2.127079e+07,
+2.730293e+07,3.120849e+07,3.200887e+07,2.767557e+07,2.010646e+07,
+1.317914e+07,8.280413e+06,4.941314e+06,2.683068e+06,1.284794e+06,
+5.362103e+05,1.967349e+05,6.721233e+04,2.575484e+04,1.448337e+04,
+1.157064e+04,1.043199e+04,9.535963e+03,8.672909e+03,7.888739e+03,
+7.238931e+03,6.745085e+03,6.396496e+03,6.161882e+03,6.001978e+03,
+5.876652e+03,5.751316e+03,5.609336e+03,5.437295e+03,5.229158e+03,
+4.984509e+03,4.706711e+03,4.401450e+03,4.075669e+03,3.736816e+03,
+),
 }
 
 ## Put values
-pileupWeight.pileupMC = pileupWeightMap["Startup2015_25ns"]
-pileupWeight.pileupRD = pileupWeightMap["Run2015_25nsSilver"]
-pileupWeight.pileupUp = pileupWeightMap["Run2015_25nsSilverUp"]
-pileupWeight.pileupDn = pileupWeightMap["Run2015_25nsSilverDn"]
+pileupWeight.pileupMC = pileupWeightMap["2016_25ns_SpringMC"]
```

Comment:
these get overwritten by https://github.com/jhgoh/CATTools/blob/facc1f6990716e8b7fd60a8a68f739cfe4857dc7/CatProducer/python/catDefinitions_cfi.py

can you updates these too